Services/Certificate/classes/Placeholder/Description/class.ilTestPlaceholderDescription.php

```diff
@@ -1,35 +1,19 @@
-<?php
+<?php declare(strict_types=1);
 /* Copyright (c) 1998-2018 ILIAS open source, Extended GPL, see docs/LICENSE */
 
 /**
  * @author  Niels Theen <ntheen@databay.de>
  */
 class ilTestPlaceholderDescription implements ilCertificatePlaceholderDescription
 {
-    /**
-     * @var ilDefaultPlaceholderDescription
-     */
-    private $defaultPlaceHolderDescriptionObject;
-
-    /**
-     * @var ilLanguage|null
-     */
-    private $language;
-
-    /**
-     * @var array
-     */
-    private $placeholder;
+    private ?ilDefaultPlaceholderDescription $defaultPlaceHolderDescriptionObject;
+    private ?ilLanguage $language;
```

Comment:
These types are not optional. Only the passed constructor arguments are optional.